docs/metamacros.md

```diff
@@ -0,0 +1,66 @@
+### Docile Metamacros
+
+Docile uses an extensible metamacro syntax within docstrings to support greater flexibility.
+
+#### The syntax constist of four required parts:
+
+* `!!` double qestionmark
+* followed by `more than 0 chars` which must be a valid julia identifier
+* `(` opening bracket - parentheses
+* `)` closing bracket - parentheses
+
```

Comment:
> and is interpreted as a string.

I skip that as it might depend on a metamacro too. (example in the future if there are nested once.